examples/yoti_example_django/Dockerfile

```diff
@@ -6,11 +6,12 @@ RUN if [ "$YOTI_SCENARIO_ID" = "yourScenarioId" ] ; then echo YOTI_SCENARIO_ID n
 RUN if [ "$YOTI_CLIENT_SDK_ID" = "yourClientSdkId" ] ; then echo YOTI_CLIENT_SDK_ID not set; exit 1; else echo YOTI_CLIENT_SDK_ID is $YOTI_CLIENT_SDK_ID ; fi
 RUN if [ "$YOTI_KEY_FILE_PATH" = "yourKeyFilePath" ] ; then echo YOTI_KEY_FILE_PATH not set; exit 1; else echo YOTI_KEY_FILE_PATH is $YOTI_KEY_FILE_PATH ; fi
 ADD . /code
-WORKDIR /code
+WORKDIR /code/examples/yoti_example_django/
+RUN pip install --no-cache-dir -r /code/requirements.txt && pip install /code 
```

Comment:
Would be good if we can rename `code` to `yoti-sdk` or something to make it clearer what it is 👍